public/index.html

```diff
@@ -10,7 +10,7 @@
     <link rel="apple-touch-icon" href="/assets/favicon/apple-touch-icon-180x180.png" sizes="180x180">
     <link rel="apple-touch-icon" href="/assets/favicon/apple-touch-icon-76x76.png"   sizes="76x76">
     <link rel="apple-touch-icon" href="/assets/favicon/apple-touch-icon-precomposed.png">
-    <link rel="manifest" href="/manifest.json" />
+    <link rel="manifest" href="manifest.json" />

```

Comment:
Just out of curiosity, how come this was needed? Both this index file and the manifest would be in the public root folder, so I would have expected `/manifest.json` to work.

Reply:
Yeah I can revert that change - I noticed I was getting a 401 Unauthorised response for manifest.json in console and was just testing that out.  It wasn't related to `/`, it seems to be staging specific - doesn't happen on local/prod.